### DISCUSSION TITLE
[EN Choice] Include more synonyms for the boolean choices

### DISCUSSION THREAD

**Discussion by xtianus79** [microsoft/Recognizers-Text#2067] [Ideas]
Currently these are the choices
```
TrueRegex: !simpleRegex
  def: \b(true|yes|yep|yup|yeah|y|sure|ok|agree)\b|(\uD83D\uDC4D|\uD83D\uDC4C|\u0001f44c)
FalseRegex: !simpleRegex
  def: \b(false|nope|nop|no|not\s+ok|disagree)\b|(\uD83D\uDC4E|\u270B|\uD83D\uDD90|\u0001F44E|\u0001F590)
```
do and do not 
I don't and I do
ok and not ok

it would be really helpful and an easy change. 

Also, would it be possible to pass in our own synonyms to define it further for our needs specifically?

**Comment by tellarin** [comment]
@xtianus79 we'll plan to add the new synonyms. But adding a feature to allow users to pass their own lists will take longer.
Would you be able to contribute PRs for these? Any contribution is greatly welcome.
